2023-07-04 13:44:00 警告 [main] org.apache.catalina.core.StandardServer.await 在调用accept()方法之后，侦听shutdown命令的套接字经历了意外的超时[79,061,242]毫秒。 这是bug 56684的一个例子？
	java.net.SocketTimeoutException: Accept timed out
		at java.base/java.net.PlainSocketImpl.socketAccept(Native Method)
		at java.base/java.net.AbstractPlainSocketImpl.accept(AbstractPlainSocketImpl.java:474)
		at java.base/java.net.ServerSocket.implAccept(ServerSocket.java:551)
		at java.base/java.net.ServerSocket.accept(ServerSocket.java:519)
		at org.apache.catalina.core.StandardServer.await(StandardServer.java:609)
		at org.apache.catalina.startup.Catalina.await(Catalina.java:864)
		at org.apache.catalina.startup.Catalina.start(Catalina.java:810)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.base/java.lang.reflect.Method.invoke(Method.java:566)
		at org.apache.catalina.startup.Bootstrap.start(Bootstrap.java:342)
		at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:473)
2023-07-04 14:34:28 警告 [main] org.apache.catalina.core.StandardServer.await 在调用accept()方法之后，侦听shutdown命令的套接字经历了意外的超时[3,027,889]毫秒。 这是bug 56684的一个例子？
	java.net.SocketTimeoutException: Accept timed out
		at java.base/java.net.PlainSocketImpl.socketAccept(Native Method)
		at java.base/java.net.AbstractPlainSocketImpl.accept(AbstractPlainSocketImpl.java:474)
		at java.base/java.net.ServerSocket.implAccept(ServerSocket.java:551)
		at java.base/java.net.ServerSocket.accept(ServerSocket.java:519)
		at org.apache.catalina.core.StandardServer.await(StandardServer.java:609)
		at org.apache.catalina.startup.Catalina.await(Catalina.java:864)
		at org.apache.catalina.startup.Catalina.start(Catalina.java:810)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.base/java.lang.reflect.Method.invoke(Method.java:566)
		at org.apache.catalina.startup.Bootstrap.start(Bootstrap.java:342)
		at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:473)
2023-07-04 15:33:45 警告 [main] org.apache.catalina.core.StandardServer.await 在调用accept()方法之后，侦听shutdown命令的套接字经历了意外的超时[3,556,704]毫秒。 这是bug 56684的一个例子？
	java.net.SocketTimeoutException: Accept timed out
		at java.base/java.net.PlainSocketImpl.socketAccept(Native Method)
		at java.base/java.net.AbstractPlainSocketImpl.accept(AbstractPlainSocketImpl.java:474)
		at java.base/java.net.ServerSocket.implAccept(ServerSocket.java:551)
		at java.base/java.net.ServerSocket.accept(ServerSocket.java:519)
		at org.apache.catalina.core.StandardServer.await(StandardServer.java:609)
		at org.apache.catalina.startup.Catalina.await(Catalina.java:864)
		at org.apache.catalina.startup.Catalina.start(Catalina.java:810)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.base/java.lang.reflect.Method.invoke(Method.java:566)
		at org.apache.catalina.startup.Bootstrap.start(Bootstrap.java:342)
		at org.apache.catalina.startup.Bootstrap.main(Bootstrap.java:473)
